File: src/main/java/xyz/acrylicstyle/hackReport/commands/PlayerCommand.java
package xyz.acrylicstyle.hackReport.commands;

import org.bukkit.ChatColor;
import org.bukkit.entity.Player;
import xyz.acrylicstyle.api.MojangAPI;
import xyz.acrylicstyle.hackReport.HackReport;
import xyz.acrylicstyle.tomeito_api.command.PlayerOpCommandExecutor;

import java.util.UUID;

public class PlayerCommand extends PlayerOpCommandExecutor {
    @Override
    public void onCommand(Player player, String[] args) {
        if (args.length == 0) {
            player.sendMessage(ChatColor.RED + "/player <player>");
            return;
        }
        String p = args[0];
        UUID uuid = MojangAPI.getUniqueId(p);
        if (uuid == null) {
            player.sendMessage(ChatColor.RED + "プレイヤーが見つかりません。");
            player.sendMessage(ChatColor.RED + "/player <player>");
            return;
        }
        player.openInventory(HackReport.PLAYER_ACTION_GUI.prepare(player, args[0], uuid).getInventory());
    }
}

File: src/main/java/xyz/acrylicstyle/hackReport/commands/PlayerCommand.kt
package xyz.acrylicstyle.hackReport.commands

import org.bukkit.ChatColor
import org.bukkit.entity.Player
import xyz.acrylicstyle.api.v1_8_R1.MojangAPI
import xyz.acrylicstyle.hackReport.gui.PlayerActionGui
import xyz.acrylicstyle.tomeito_api.command.PlayerCommandExecutor

class PlayerCommand : PlayerCommandExecutor() {
    override fun onCommand(player: Player, args: Array<String>) {
        if (args.isEmpty()) {
            player.sendMessage(ChatColor.RED.toString() + "/player <player>")
            return
        }
        val p = args[0]
        val uuid = MojangAPI.getUniqueId(p)
        if (uuid == null) {
            player.sendMessage(ChatColor.RED.toString() + "プレイヤーが見つかりません。")
            player.sendMessage(ChatColor.RED.toString() + "/player <player>")
            return
        }
        player.openInventory(PlayerActionGui().register().prepare(player, args[0], uuid).inventory)
    }
}